app/controllers/item.js

```diff
@@ -41,6 +42,11 @@ export default Controller.extend({
       : [item];
   }),
 
+  notAvailableInStock: computed("allPackages.@each.quantity", function() {
+    let quantities = this.get("allPackages").map(pkg => pkg.get("quantity"));
+    return _.sum(quantities) === 0;
+  }),
+
```

Comment:
We're in the process of removing the `quantity` field, as it is faulty. So if you could avoid using it that'd be very helpful :)
As of now the available quantity can still be inferred from the `packages_locations` and the `orders_packages`.

In the near future we will have multiple fields like `available_quantity`, `designated_quantity`, etc... (IN PROGRESS)

Reply:
@swatijadhav , you can check the `stockit_item_serializer`(`on_hand_attribute`) to check how we're calculating the available quantity. 

Unfortunately, we are not using `stockit_item_serializer` on browse. So you can check how we can send the `on_hand_quantity` to browse using the `package_serializer`. 

Using quantity from now on can become misleading in the coming future and we might need to fix that throughout. As @patrixr mentioned , we want to get rid of this quantity column.

Reply:
If the records are available on the frontend then a computed property could be enough as a temporary measure, rather than making more api changes